### PR TITLE
Map fixes and styling

### PR DESCRIPTION
**Pull request by sandreae**
- Fix error with incorrectly formatted virtual events array
- Style the map popup nicely

### CODE REVIEW

File: app/styles/vendors/react-leaflet.scss [hunk 1]
```diff
@@ -554,6 +554,9 @@
 }
 
 .leaflet-popup-content {
+
```
Comment (adzialocha) [inline app/styles/vendors/react-leaflet.scss:557]: Miau

File: app/styles/vendors/react-leaflet.scss [hunk 2]
```diff
@@ -545,7 +545,7 @@
   text-align: center;
 }
 
-.leaflet-popup-content-wrapper {
+.leaflet-popup-content-wrapper {
```
Comment (adzialocha) [inline app/styles/vendors/react-leaflet.scss:548]: Wuff

File: app/styles/components/map-popup.scss [hunk 3]
```diff
@@ -1,11 +1,21 @@
-.map-popup-img {
-  display: block;
-
-  max-width: 80px;
-
-  margin: 0 auto;
+.map-popup-event-list {
+  max-height: 250px;
+  width: 100%;
+  overflow-y: scroll;
```
Comment (adzialocha) [inline app/styles/components/map-popup.scss:4]: Blubb

**Review by adzialocha — CHANGES_REQUESTED** [review]
Looks like the linter is not happy :cry:

**Comment by sandreae** [comment]
I didn't realize `stylelint` was in the mix.....

**Review by adzialocha — APPROVED** [review]

**Pull request merged**